navigation/app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/NavigationViewActivity.java

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.services.android.navigation.testapp.activity.navigationui;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+import com.mapbox.services.android.navigation.testapp.R;
+import com.mapbox.services.android.navigation.ui.v5.NavigationView;
+import com.mapbox.services.android.navigation.v5.navigation.NavigationConstants;
+
+public class NavigationViewActivity extends AppCompatActivity {
+
+  @Override
+  protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_navigation_view);
+
+    Button launchNavigationBtn = (Button) findViewById(R.id.launchBtn);
+    launchNavigationBtn.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        launchNavigation();
+      }
+    });
+  }
+
+  private void launchNavigation() {
+    Intent navigationView = new Intent(this, NavigationView.class);
```

Comment:
Are we settling on calling this a `NavigationView`? Isn't it technically an activity? Just making sure this is the name we want to go with.